x/ccv/provider/keeper/keeper.go

```diff
@@ -293,6 +293,17 @@ func (k Keeper) SetConsumerChain(ctx sdk.Context, channelID string) error {
 	k.SetInitChainHeight(ctx, chainID, uint64(ctx.BlockHeight()))
 	// - remove init timeout timestamp
 	k.DeleteInitTimeoutTimestamp(ctx, chainID)
+
+	// emit event on successful addition
+	ctx.EventManager().EmitEvent(
+		sdk.NewEvent(
+			ccv.EventTypeConsumerChainAdded,
```

Comment:
Change this event to `EventTypeCCVChannelEstablished`.

Reply:
`Done`.